activerecord/lib/active_record/railtie.rb

```diff
@@ -26,6 +26,9 @@ class Railtie < Rails::Railtie # :nodoc:
     config.active_record.use_schema_cache_dump = true
     config.active_record.maintain_test_schema = true
 
+    config.active_record_sqlite3adapter = ActiveSupport::OrderedOptions.new
+    config.active_record_sqlite3adapter.represent_boolean_as_integer = nil
```

Comment:
The `set_configs` initializer targets all values under `app.config.active_record`.
https://github.com/rails/rails/blob/master/activerecord/lib/active_record/railtie.rb#L111..L113 

Therefore, `config.active_record.sqlite3` is also subject to the set, so it is necessary to delete `sqlite3` as before. 

Or can check with `respond_to` when `send` with `set_configs`, but there is a slightly redundant feeling...